src/Momento.Sdk/Config/Retry/IGrpcRequest.cs

```diff
@@ -0,0 +1,6 @@
+namespace Momento.Sdk.Config.Retry;
+
+public interface IGrpcRequest
```

Comment:
curious if you are envisioning a need for a wrapper here that encapsulates the actual grpc classes?

this is 100% totally fine for this PR but I wasn't sure if we would end up needing a wrapper for these.  can decide later.

Reply:
I put these here as stubs to move forward. I will use the appropriate gRPC base types/interface if there is one.

Reply:
np this is great for now, just checking